docs/4-querying-lists.md

```diff
@@ -17,6 +17,21 @@ const queryObservable = af.database.list('/items', {
 });
 ```
 
+**Query Options:**
+
+| method   | purpose            | 
+| ---------|--------------------| 
+| orderByChild | specify a child to order by |
+| orderByKey | boolean to order by Firebase keys |
+| orderByPriority | boolean to order by Firebase priority |
+| orderByValue | specify a value to order by |
+| equalTo | limit list to items that contain certain value |
+| limitToFirst | limit response to x items starting at the beginning |
+| limitToLast | limit response to x items starting at the end |
```

Comment:
Avoid using the word "response", as there is no HTTP Request/Response mechanism going on. 

Suggestion for `limitToFirst`: "Sets the maximum number of items to return from the beginning of the ordered list of results."